logfire/_internal/integrations/redis.py

```diff
@@ -38,9 +38,10 @@ def instrument_redis(capture_statement: bool = False, **kwargs: Unpack[RedisInst
 
 
 def _capture_statement_hook(request_hook: RequestHook | None = None) -> RequestHook:
-    def _capture_statement(span: Span, instance: Connection, *args: Any, **kwargs: Any) -> None:
-        command = ' '.join(args[0])
-        span.set_attribute('db.statement', command)
+    def _capture_statement(
+        span: Span, instance: Connection, command: tuple[str, ...], *args: Any, **kwargs: Any
+    ) -> None:
+        span.set_attribute('db.statement', ' '.join(command))
```

Comment:
1. Please check that `command` really only ever contains strings, i.e. that `client.set("my-key", 123)` won't break here.
2. Please set the logfire message to a similar value but with `truncate_string(s, max_length=20, middle='…')` on each part, so that each part is easily visible even when there are big keys/values.